{{ cookiecutter.__project_name_kebab }}/pyproject.toml

```diff
@@ -27,30 +27,27 @@ classifiers = [
     "Operating System :: OS Independent",
     "Programming Language :: Python",
     "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.7",
     "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Framework :: Django",
-    "Framework :: Django :: 3.0",
-    "Framework :: Django :: 3.1",
     "Framework :: Django :: 3.2",
-    "Framework :: Django :: 4.0",
     "Framework :: Django :: 4.1",
+    "Framework :: Django :: 4.2",
     "Framework :: Wagtail",
-    "Framework :: Wagtail :: 2",
-    "Framework :: Wagtail :: 4",
+    "Framework :: Wagtail :: 4.1",
+    "Framework :: Wagtail :: 4.2",
+    "Framework :: Wagtail :: 5.0",
 ]
 dynamic = ["version"]
 dependencies = [
-    "Django>=3.0,<4.2",
-    "Wagtail>=2.15,<5.0"
+    "Wagtail>=4.1,<5.1"
 ]
 [project.optional-dependencies]
 testing = [
-    "dj-database-url==1.2.0",
-    "pre-commit>=2.21.0,<3.0"
+    "dj-database-url>=2.0.0",
+    "pre-commit"
```

Comment:
The reason this was pinned to `pre-commit<3.0` is because the cookiecutter generates a package which supports `python>=3.7`. pre-commit 3.0 requires `python>=3.8`. If we're keeping 3.7 compatibility, we need to retain that constraint.